error-codes.md

```diff
@@ -133,102 +139,96 @@ In addition, TiDB has the following unique error codes:
 
     The field fails to obtain the default value. This error is usually used internally, and is converted to a specific type of error for external applications.
 
-* Error Number: 8039
-
-    Index offset is out of range.
-
-* Error Number: 8042
-
-    The state of table schema does not exist.
-
-* Error Number: 8043
-
-    The state of column information does not exist.
-
-* Error Number: 8044
-
-    The state of index does not exist.
-
-* Error Number: 8045
-
-    Invalid table data.
+* Error Number: 8040
 
-* Error Number: 8046
-
-    The state of column information is invisible.
+    Unsupported operations are performed. For example, operating lock table on View and Sequence.
 
 * Error Number: 8047
 
     The value of the system variable is not supported. This error usually occurs in the alarm information when the user sets a variable value that is not supported in the database.
 
 * Error Number: 8048
 
-    An unsupported database isolation level is set.
+    An unsupported database isolation level is set. 
+    
+    If you cannot modify the codes by using third-party tools or frameworks, use `tidb_skip_isolation_level_check` to bypass this check.
```

Comment:
```suggestion
    If you cannot modify the codes because you are using a third-party tool or framework, consider using `tidb_skip_isolation_level_check` to bypass this check.
```